### PR TITLE
House Filter: refactor profession filtering

### PR DESCRIPTION
**Pull request by shlior7**
this way its filtering the profession of either parent and also makes this looks cleaner

### CODE REVIEW

**Comment by adiabramovitch** [comment]
Can you add tests?

**Review by LiorKGOW — CHANGES_REQUESTED** [review]
Nice idea @shlior7 👍 
I'll approve after adding a few tests
Maybe you could activate the filter with a profession firstly with p1 and then with p2
and after getting the data, making sure they filter the same data
or
checking the both of them indivisualy without comparrison - Successful find
or
the same but unsuccessful find

**Review by ShellyGolden — APPROVED** [review]

File: tests/house_tests.py [hunk 1]
```diff
@@ -83,6 +83,24 @@ def test_filter_lowest_income(self):
...
+        ).save()
+        HouseFactory(
+            country=Country.objects.get(name='Israel'),
+            city=City.objects.get(name='Tel Aviv'),
+            income=5000,
+            children=1,
+            parent_profession_1=Job.TEACHER,
+        ).save()
+        houses = _filter_houses_by_form(form_data_filter_tests(), House.objects.all())
+        assert len(houses) == 2
```
Comment (adiabramovitch) [inline tests/house_tests.py:102]: Can you add an assert which checks if the professions match the filter?

**Comment by adiabramovitch** [comment]
Can you squash the commits?
Plz, change the commit-msg to be more specific about what did you add in this PR.

File: tests/house_tests.py (hunk 1 above)
Comment (shlior7) [reply tests/house_tests.py:102]: ok I now added an assert that checks that Teacher is in profession 1 and 2 and in houses meaning was not filtered

**Pull request merged**